twake/backend/node/src/services/user/web/schemas.ts

```diff
@@ -1,20 +1,28 @@
-export const createUserSchema = {
-  body: {
-    type: "object",
-    properties: {
-      email: {
-        type: "string",
-      },
-    },
-    required: ["email"],
-  },
+export const userSchema = {
+  type: "object",
+  properties: {
+    id: { type: "string" },
+    connected: { type: "boolean" },
+    creationdate: { type: "string" },
+    firstname: { type: "string" },
+    lastname: { type: "string" },
+    usernamecanonical: { type: "string" },
+    emailcanonical: { type: "string" },
+    phone: { type: "string" },
+    timezone: { type: "string" },
+    groups: { type: "array" },
```

Comment:
Here we should use a better schema for the future, I will document this.
At least we could replace groups by companies ?

Reply:
Yes and no :)
I just want to map the current schema. When import will be OK and when we will start using the node backend to manage companies, we will have a new, cleaner schema, and cleaner fields, and cleaner parameter names